README.md

```diff
@@ -164,7 +163,7 @@ releases][olderusage] on the shapeless wiki.
 
 ## Building shapeless
 
-shapeless is built with SBT 1.3.10 or later, and its master branch is built with Scala 2.13.2 by default but also
+shapeless is built with SBT 1.3.10 or later, and its main branch is built with Scala 2.13.2 by default but also
 cross-builds for 2.11.12 and 2.12.12.
```

Comment:
I don't know if we need to mention concrete versions, they are always out of date

Reply:
I'll pick that up later.